src/qt/rpcconsole.cpp

```diff
@@ -731,8 +731,12 @@ void RPCConsole::clear(bool clearHistory)
         );
 
     message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(tr(PACKAGE_NAME)) + "<br>" +
-                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +
-                        tr("Type <b>help</b> for an overview of available commands.")) +
+#ifdef Q_OS_MAC
+                        tr("Use up and down arrows to navigate history, and <b>⌘cmd-L</b> to clear screen.") + "<br>" +
+#else
+                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +		
+#endif
+			tr("Type <b>help</b> for an overview of available commands.")) +
```

Comment:
Yes, adding it to platformStyle would be another step further, I didn't want to suggest that here to not make life too difficult for @spencerlievens while also not making life harder for translators by adding another message that is almost-the-same.